sources/nodes/types.go

```diff
@@ -30,6 +30,10 @@ type Info struct {
 	InternalIP string
 	// An optional ID assigned to nodes by either the cloud provider or user.
 	ExternalID string
+	// Cpu capacity of node as number of 1/1024 share of core.
+	CpuCapacity uint64
```

Comment:
Just wanted to make it consistent with the current unit, but it will be definitely easier to use millicores instead.